conans/test/unittests/client/rest/uploader_test.py

```diff
@@ -29,13 +31,15 @@ def put(self, *args, **kwargs):
     def test_403_raises_unauthoirzed_exception_if_no_token(self):
 
         class MockRequester(object):
+            retry = 0
```

Comment:
Is this needed in the current implementation? Are the retry finally members of the requester?